typings/alt/alt.d.ts

```diff
@@ -0,0 +1,158 @@
+// Type definitions for Alt 0.16.7
+// Project: https://github.com/goatslacker/alt
+// Definitions by: Michael Shearer <https://github.com/Shearerbeard>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../flux/flux.d.ts"/>
+/// <reference path="../eventemitter3/eventemitter3.d.ts"/>
+
+
+declare module AltJS {
+
+  export interface StoreModel<S> {
+    bindActions?( ...actions:Array<Object>);
+    exportPublicMethods?<M>(exportConfig:M):void;
+    getState?():S;
+    exportAsync?(source:Source);
+    waitFor?(store:AltStore<any>):void;
+  }
+
+  export type Source = {[name:string]:() => SourceModel};
+
+  export interface SourceModel {
+    local( ...args:Array<any>);
+    remote( ...args:Array<any>);
+    shouldFetch?(state:Object, ...args:Array<any>);
+    loading: ( ...args:Array<any>) => void;
+    success:( ...args:Array<any>) => void;
+    error:( ...args:Array<any>) => void;
+    interceptResponse?(response:Object, action:AltJS.Action<any>, ...args:Array<any>);
+  }
+
+  export interface AltStore<S> {
+    getState():S;
+    listen(handler:(state:S) => any):() => void;
+    unlisten(handler:(state:S) => any):void;
+    emitChange():void;
+    getEventEmitter():EventEmitter3.EventEmitter;
```

Comment:
this is actually removed in next version (currently master), should we move forward without it or support two d.ts files?